test/standalone/test-debuglet.js

```diff
@@ -48,6 +48,7 @@ describe(__filename, function() {
   });
 
   it('should not start when projectId is not available', function(done) {
+    this.timeout(8000);
```

Comment:
I think we should optionally disable retry in `gcp-metadata`, but I don't think it is required for this PR.